### PR TITLE
Feature/remove byte array

### PR DESCRIPTION
**Pull request by hrcornejo**

### CODE REVIEW

File: src/main/groovy/org/ods/doc/gen/core/ZipFacade.groovy [hunk 1]
```diff
@@ -14,15 +14,17 @@ class ZipFacade {
...
         String path = "${tmpFolder}/${ARTIFACTS_BASE_DIR}/${name}".toString()
         Files.createDirectories(Paths.get(path).parent)
 
         def zipFile = new ZipFile(path)
-        files.each { filePath, fileData ->
+        files.each { fileName, filePath ->
             def params = new ZipParameters()
-            params.setFileNameInZip(filePath)
-            zipFile.addStream(new ByteArrayInputStream(fileData), params)
+            params.setFileNameInZip(fileName)
```
Comment (victorpablosceruelo) [inline src/main/groovy/org/ods/doc/gen/core/ZipFacade.groovy:24]: Cant FIS be obtained without using File ?

File: src/main/groovy/org/ods/doc/gen/core/api/HealthController.groovy [hunk 2]
```diff
@@ -17,42 +17,39 @@ class HealthController {
...
-                time: new Date().toString()
+                status : "passing",
+                time   : new Date().toString()
         ]
 
         return result
     }
 
-    private byte[] generatePdfData() {
+    private generatePdfData() {
```
Comment (victorpablosceruelo) [inline src/main/groovy/org/ods/doc/gen/core/api/HealthController.groovy:37]: void ?

**Review by victorpablosceruelo — CHANGES_REQUESTED** [review]
Please run in a Box and tell us about result.

File: src/main/groovy/org/ods/doc/gen/core/api/HealthController.groovy (hunk 2 above)
Comment (hrcornejo) [reply src/main/groovy/org/ods/doc/gen/core/api/HealthController.groovy:37]: Yes, is a Health controller and we weren't using the result.

File: src/main/groovy/org/ods/doc/gen/core/ZipFacade.groovy (hunk 1 above)
Comment (hrcornejo) [reply src/main/groovy/org/ods/doc/gen/core/ZipFacade.groovy:24]: Nop

Comment (hrcornejo) [reply src/main/groovy/org/ods/doc/gen/core/ZipFacade.groovy:24]: Nop

**Comment by s2oBCN** [comment]
> Please run in a Box and tell us about result.

Why do you need to run it in a Box? What is the change in the  docker service when you run it in a box?

**Comment by hrcornejo** [comment]
> Please run in a Box and tell us about result.

@victorpablosceruelo seems that everything works:
image
image
All the documents are generated and uploaded in Nexus

**Review by victorpablosceruelo — APPROVED** [review]

**Pull request merged**